### DISCUSSION TITLE
I want to upload files from old andorid phone like oneplus one that dont support immich app

### DISCUSSION THREAD

**Discussion by sumitshrestha** [immich-app/immich#9428] [General]
I’ve successfully set up Immich on my local server and am enjoying the process of uploading photos from my iPhone and Samsung phone. However, I encountered a snag when attempting to install Immich on my vintage OnePlus One; the Play Store indicated that the app is not compatible with my device. Even my attempt to install it via an APK file was unsuccessful. Given these challenges, I’m curious to know if there’s an available API that would allow me to manually upload photos by developing a custom Android application.

**Comment by schuhbacca** [comment]
Immich has an api that can be found here: https://immich.app/docs/api/

You can also try compiling the mobile app to an earlier android sdk version. The version of the app on the play store is the minimum acceptable android target to be in compliance with play store policy from what I remember. 

Either way there are lots of avenues that can be taken to get them off an older device.
